### ISSUE TITLE
Persistent sessions name limitations

### ISSUE THREAD

**Issue by ccarouge** [ACCESS-NRI/access-hive.org.au#796]
## Description

I've just started my first persistent session today. I tried calling it "test_199b" and got:
```
$ persistent-sessions start test_199b
[  1726793193.092319] [CRITICAL] [nci_launch] [client/source/launch.c:449] invalid session name: test_199b
```

It seems persistent session names have limitations on valid characters, it might be good to warn users about it in the instructions.

I don't know what the limitations are, it might be only alpha-numeric characters...

## Close Criteria
Information in the instructions about valid characters for persistent session names.

**Comment by atteggiani** [comment]
Hi @ccarouge, 

Thank you for submitting this issue. 

I agree, there are some things with `persistent-sessions` that are not optimal. 
Along with the problem you pointed out, another issue is that only the session ID gets printed out when running `persistent-sessions list`, but no name. 
This is always very frustrating for me, because if I want to verify whether a persistent session with a specific name is running, I have to manually check in the `~/.persistent-session` folder for the ID associated with the name of the session, and then check that that is the ID of the running session.

Since I don't know the exact current limitations of `persistent-sessions` (and I cannot find information about it on the NCI website), I would not add any information on the Hive Dos either. 

But I think the `persistent-sessions` functionality could be improved (both on what you pointed out and on my issue), which is something we were already talking about with @penguian. 
He already spoke to Ruy Yang at NCI and we might have a meeting with him to discuss these improvements.

Does it make sense to you?

**Comment by ccarouge** [comment]
Yes, it makes sense to me. I'm not sure I agree with not warning users of the limitations we already know but that's not my call.

**Comment by atteggiani** [comment]
Yes, I understand your concerns. 

The issue is that I don't know what limitations on characters are in place. 
What is your suggested text for a warning on the characters limitation?

**Comment by ccarouge** [comment]
I would put a warning against using any special character and using a single "word" name, CamelCase can be used to separate words. Example:
---
Persistent session names accept a limited set of characters. We recommend using only alpha-numeric characters without any space in the chosen name. CamelCase can be used to identify separate words within the name. E.g:
✅ MyTest
✅ EnsMember3
❌ My_Test
❌ my test

**Comment by atteggiani** [comment]
@ccarouge would you happy to create a PR for this?

**Comment by atteggiani** [comment]
We're addressing this in #1033
